Makefile

```diff
@@ -1,13 +1,19 @@
 .DEFAULT_GOAL := help
 
+GO_BIN ?= $(shell go env GOPATH)/bin
+
 .PHONY: help
 help:
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
+$(GO_BIN)/golangci-lint:
+	@echo "==> Installing golangci-lint within "${GO_BIN}""
+	@go install -v github.com/golangci/golangci-lint/cmd/golangci-lint@latest
+
 .PHONY: lint
-lint: ## Lint Go source files
-	@go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest
-	@golangci-lint run
+lint: $(GO_BIN)/golangci-lint ## Lint Go source files
+	@echo "==> Linting Go source files"
+	@golangci-lint run -v --fix -c .golangci.yaml ./...
```

Comment:
Ensuring we run make lint with the `--fix` flag now to fix all the imports for us with one command.